### PR TITLE
feat(HMS-4520): aggregate identity information

### PR DESCRIPTION
**Pull request by avisiedo**
This change aggregate the org_id and the identity
information to the contextual log, so the information
is printed out for any log message.

### CODE REVIEW

File: internal/infrastructure/middleware/enforce_identity.go [hunk 1]
```diff
@@ -13,6 +14,15 @@ import (
 	identity "github.com/redhatinsights/platform-go-middlewares/v2/identity"
 )
 
+// https://github.com/RedHatInsights/identity-schemas/blob/main/3scale/schema.json#L21
+const (
+	identityTypeUser          = "User"
+	identityTypeServieAccount = "ServiceAccount"
```
Comment (pvoborni) [inline internal/infrastructure/middleware/enforce_identity.go:20]: Typo: `identityTypeServieAccount` vs `identityTypeServiceAccount`

File: internal/infrastructure/middleware/enforce_identity.go [hunk 2]
```diff
@@ -13,6 +14,15 @@ import (
 	identity "github.com/redhatinsights/platform-go-middlewares/v2/identity"
 )
 
+// https://github.com/RedHatInsights/identity-schemas/blob/main/3scale/schema.json#L21
```
Comment (pvoborni) [inline internal/infrastructure/middleware/enforce_identity.go:17]: Line 21 points to `auth_type` and not `type`. Also `type` there has only 3 values - `User`, `ServiceAccount`, `System`. 

The: `X509` and `Associate` are bit weird. It's coming from  in https://github.com/RedHatInsights/platform-go-middlewares/blob/master/identity/identity.go#L101 right?  I also saw them used as type in https://github.com/RedHatInsights/platform-go-middlewares/blob/master/identity/identity_suite_test.go so I guess we should rather work with them - so OK.

**Review by pvoborni — COMMENTED** [review]
Mostly looks good to me, didn't test yet.

File: internal/infrastructure/middleware/enforce_identity.go (hunk 1 above)
Comment (avisiedo) [reply internal/infrastructure/middleware/enforce_identity.go:20]: thanks! Updated!

File: internal/infrastructure/middleware/enforce_identity.go (hunk 2 above)
Comment (avisiedo) [reply internal/infrastructure/middleware/enforce_identity.go:17]: good catch! I have updated the link! thanks!

Comment (avisiedo) [reply internal/infrastructure/middleware/enforce_identity.go]: you are right; `X509` and `Associate` are comming from the link you referenced; I added them as I have seen in the json schema definition at: https://github.com/RedHatInsights/identity-schemas/blob/main/turnpike/schema.json just for considering all the potential values and reflect the mapping in the log, and avoid `unknown` for them.

**Review by pvoborni — APPROVED** [review]
The PR LGTM, it works.  

How do we restart the CI job?

**Pull request merged**